src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_FOR_RENT = 40;
+
+  const AMOUN_DAY_FOR_MIN_DISCOUNT = 3;
+  const AMOUN_DAY_FOR_MAX_DISCOUNT = 7;
```

Comment:
There is a typo in the variable names 'AMOUN_DAY_FOR_MIN_DISCOUNT' and 'AMOUN_DAY_FOR_MAX_DISCOUNT'. It should be 'AMOUNT_DAY_FOR_MIN_DISCOUNT' and 'AMOUNT_DAY_FOR_MAX_DISCOUNT' to correctly represent the meaning and to avoid confusion.